pkg/chartmuseum/server/multitenant/server.go

```diff
@@ -157,6 +159,12 @@ func NewMultiTenantServer(options MultiTenantServerOptions) (*MultiTenantServer,
 		TenantCacheKeyLock:     &sync.Mutex{},
 		CacheInterval:          options.CacheInterval,
 		ChartLimits:            l,
+		WebTemplatePath:        options.WebTemplatePath,
+	}
+
+	if server.WebTemplatePath != "" {
+		// load html templates
+		server.Router.LoadHTMLGlob(fmt.Sprintf("%s/*.html", server.WebTemplatePath))
```

Comment:
This will panic if the directory/files don't exist. I'm wondering if the service should continue on even if the files don't exist 🤔

Reply:
That might be good since ChartMuseum can function without the static files/web pages.

Reply:
Done, looking for your feedback.

Reply:
Looks good!